Description: Remove external image links for paypal
 donation (lintian E: privacy-breach-donation)
Author: Sergey B Kirpichev <skirpichev@gmail.com>
Forwarded: no

---
 docs/index.html |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/docs/index.html
+++ b/docs/index.html
@@ -152,8 +152,7 @@
 <form action="https://www.paypal.com/cgi-bin/webscr" method="post">
 <input type="hidden" name="cmd" value="_s-xclick">
 <input type="hidden" name="hosted_button_id" value="8968579">
-<input type="image" src="https://www.paypal.com/en_US/i/btn/btn_donateCC_LG_global.gif" border="0" name="submit" alt="PayPal - The safer, easier way to pay online.">
-<img alt="" border="0" src="https://www.paypal.com/en_US/i/scr/pixel.gif" width="1" height="1">
+<input type="image" border="0" name="submit" alt="Donate with PayPal">
 </form>
 <!-- END_PAYPAL_LINK -->
 
